Automattic/wp-calypso
# A page selector component

#### Changes proposed in this Pull Request

The UI Shell required for the page selector component.

![pageselection](https://user-images.githubusercontent.com/3422709/158765463-ab8f0de6-8f25-482c-9591-1f4a21a7bcc2.gif)



#### Testing instructions

- Go to /start/do-it-for-me/?flags=signup/redesigned-difm-flow
- Confirm that the UI matches the screenshot shared above.
